File: result/translate/starchat/python2java/java/FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7.java
public static int f_gold(String n) {
        int i = 0, j = n.length();
        int pos = 0;
        while (i < j) {
            if (n.charAt(i) == '4') {
                pos = pos * 2 + 1;
            } else if (n.charAt(i) == '7') {
                pos = pos * 2 + 2;
            }
            i++;
        }
        return pos;
    }

File: result/translate/starchat/java2python/python/FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7.py
def f_gold(n):
        k = 0
        pos = 0
        i = 0
        while k!= len(n):
            if n[i] == '4':
                pos = pos * 2 + 1
            elif n[i] == '7':
                pos = pos * 2 + 2
            i += 1
            k += 1
        return pos